import_export_extensions/admin/model_admins/export_job_admin.py

```diff
@@ -211,5 +214,19 @@ def get_fieldsets(
 
         return [status, traceback_fields, export_params]
 
+    @admin.action(description="Cancel selected jobs")
+    def cancel_jobs(self, request, queryset):
+        """Admin action for cancelling data export."""
+        for obj in queryset:
+            try:
+                obj.cancel_export()
+                self.message_user(
+                    request,
+                    _("Export Canceled"),
```

Comment:
Let's also add representation of failed `obj` to messages, ex:
"Export of {str(obj)} Canceled"